CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ First of all, thanks for taking the time to contribute! 🎉👍
 - [Practices](#practices)
   - [Errors handling](#errors-handling)
 
----
+- - -
```

Comment:
I personally prefer using the style with spaces for horizontal ruler, because it is unambiguous with the underline title style:

```markdown
This is not a title.

---
This is under a ruler.
```

is less clear to me than

```markdown
This is not a title.

- - -
This is under a ruler.
```

and in case of a forgotten whitespace, this will still render as a ruler:

```markdown
This is not a title.
- - -
This is under a ruler.
```

whereas this won't:

```markdown
This is not a title (but is rendered as such)
---
This is under a ruler.
```

- - -

As for rule MD035, we can define the string to whatever we want, so if you agree with my point of view we could very well define it to `- - -` 🙂